RubeesEat/Model/InMemoryBillRepository.cs

```diff
@@ -28,6 +28,14 @@ public IReadOnlyList<Bill> GetAll()
             : null;
     }
 
+    public BalanceChange GetBalanceChangesForUser(List<BalanceChange> balanceChanges, Bill bill, Person currentUser)
```

Comment:
Parameter List<BalanceChange> balanceChanges is not used in function and can be removed